src/tm_mad/ceph/clone

```diff
@@ -78,6 +79,19 @@ CEPH_CONF="${XPATH_ELEMENTS[j++]}"
 SIZE="${XPATH_ELEMENTS[j++]}"
 ORIGINAL_SIZE="${XPATH_ELEMENTS[j++]}"
 
+#-------------------------------------------------------------------------------
+# Get Datastore information
+#-------------------------------------------------------------------------------
+
+unset i j XPATH_ELEMENTS
+
+while IFS= read -r -d '' element; do
+        XPATH_ELEMENTS[i++]="$element"
+      done < <(onedatastore show -x $DST_DS_ID | $XPATH \
+                            /DATASTORE/TEMPLATE/EC_POOL)
+
+EC_POOL="${XPATH_ELEMENTS[j++]}"
+
```

Comment:
No need if `INHERIT_DATASTORE_ATTR  = "EC_POOL"` is added to `oned.conf`

Reply:
I have some issues with the change `INHERIT_DATASTORE_ATTR`:

The good:
It will be faster, since it won't need to call additional `onedatastore show`
Configuration options will be simpler

The bad:
In my original implementation EC_POOL setting is taken into account separately for  `image datastore` and `system datastore`, so it allows the following combinations
- you can have `regular replicated` images in `image datastore`, which can be deployed as `erasure coding` image to `system datastore`
- you can have `erasure coding` images in `image datastore`, which can be deployed as  `regular replicated` images in `system datastore`

The ugly:
Your proposed solution with `INHERIT_DATASTORE_ATTR` will always respect only `image datastore` EC_POOL and use it on `system datastore`

Which options suits you better?

Reply:
I stand corrected. Your proposal makes more sense. Thanks for taking the time to explain it to me.